dae/dae/variants_loaders/vcf/tests/test_vcf_loader.py

```diff
@@ -247,8 +232,8 @@ def test_vcf_denovo_mode(
     }
     vcf_loader = VcfLoader(
         families,
-        [f"{prefix}.vcf"],
-        gpf_instance_2013.reference_genome,
+        [f"{str(inheritance_trio_denovo_omission_vcf)}"],
```

Comment:
This doubles conversion to string. `str(inheritance_trio_denovo_omission_vcf)` is good enough.